com.synology.CloudStationBackup.yml

```diff
@@ -0,0 +1,51 @@
+app-id: com.synology.CloudStationBackup
+default-branch: stable
+runtime: org.freedesktop.Sdk
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: synology-cloud-station-backup
+tags: [proprietary]
+finish-args:
+  - --filesystem=home
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --env=LD_LIBRARY_PATH=/app/lib
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: synology-cloud-station-backup
+    buildsystem: simple
+    build-commands:
+      - install -D synology-cloud-station-backup.sh /app/bin/synology-cloud-station-backup
+      - install -Dm644 com.synology.CloudStationBackup.desktop -t /app/share/applications
+      - install -Dm644 com.synology.CloudStationBackup.appdata.xml -t /app/share/metainfo
+      - install -Dm644 sico_72_cloud_station.png /app/share/icons/hicolor/64x64/apps/com.synology.CloudStationBackup.png
+      - install -D apply_extra -t /app/bin
+      - cp /usr/bin/ar /app/bin
+      - ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so
+        /app/lib
+      - ARCH_TRIPLE=$(gcc --print-multiarch) && ln -s /usr/lib/${ARCH_TRIPLE}/libtinfo.so/app/lib/libtinfo.so.5
+    sources:
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - ar x synology-CloudStationBackup.deb
+          - tar xf data.tar.xz
+          - mv opt/Synology/CloudStationBackup CloudStationBackup
+          - rm -r synology-CloudStationBackup.deb control.tar.gz data.tar.xz debian-binary usr
+      - type: file
+        path: synology-cloud-station-backup.sh
+      - type: file
+        path: com.synology.CloudStationBackup.desktop
+      - type: file
+        path: com.synology.CloudStationBackup.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
You're saying the file should be 
`com.synology.CloudStationBackup.metainfo.xml`
?

Reply:
Yes

Reply:
This should be resolved now.